internal/commands/result.go

```diff
@@ -322,6 +353,13 @@ func convertCxResultsToSarif(results *wrappers.ScanResultsCollection) *wrappers.
 	return sarif
 }
 
+func convertCxResultsToSonar(results *wrappers.ScanResultsCollection) *wrappers.ScanResultsSonar {
+	var sonar = new(wrappers.ScanResultsSonar)
+	sonar.Results = []wrappers.SonarIssues{}
+	sonar.Results = createSonarRun(results)
```

Comment:
createSonarRun returs parseResultsSonar
could we simplify this to sonar.Results = parseResultsSonar(results) and remove createSonarRun?

Reply:
Yes it will be more simple